app/build.gradle

```diff
@@ -90,36 +91,38 @@ if (build_param != "dev") {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
+    implementation 'androidx.multidex:multidex:2.0.1'
+
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
-    implementation 'com.google.android.material:material:1.2.0'
+    implementation 'com.google.android.material:material:1.2.1'
     implementation 'com.github.jkwiecien:EasyImage:1.3.1'
-    implementation 'com.facebook.device.yearclass:yearclass:2.0.0'
-    implementation 'com.squareup.retrofit2:retrofit:2.4.0'
-    implementation 'com.squareup.retrofit2:converter-gson:2.4.0'
-    implementation 'com.squareup.retrofit2:adapter-rxjava:2.4.0'
-    implementation 'com.squareup.okhttp3:logging-interceptor:3.11.0'
-    implementation 'io.reactivex:rxjava:1.3.0'
+    implementation 'com.facebook.device.yearclass:yearclass:2.1.0'
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
+    implementation 'com.squareup.retrofit2:adapter-rxjava:2.9.0'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.9.0'
+    implementation 'io.reactivex:rxjava:1.3.8'
     implementation 'com.journeyapps:zxing-android-embedded:3.6.0'
     implementation 'com.jakewharton:butterknife:10.2.3'
     implementation 'com.jakewharton.timber:timber:4.7.1'
-    implementation 'com.google.dagger:dagger:2.28.3'
+    implementation 'com.google.dagger:dagger:2.29.1'
     implementation 'com.squareup:otto:1.3.8'
     implementation 'org.parceler:parceler-api:1.1.13'
     implementation 'com.github.tajchert:nammu:1.2.1'
-    implementation 'com.google.firebase:firebase-core:17.5.0'
-    implementation 'com.google.firebase:firebase-crashlytics:17.2.1'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
+    implementation 'com.google.firebase:firebase-core:17.5.1'
+    implementation 'com.google.firebase:firebase-crashlytics:17.2.2'
+    implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
     annotationProcessor 'com.jakewharton:butterknife-compiler:10.2.3'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.28.3'
+    annotationProcessor 'com.google.dagger:dagger-compiler:2.29.1'
     annotationProcessor 'org.parceler:parceler:1.1.13'
     compileOnly 'javax.annotation:jsr250-api:1.0'
 
-    testImplementation 'junit:junit:4.12'
-    testImplementation 'org.mockito:mockito-core:1.+'
-    testImplementation 'org.robolectric:robolectric:4.1'
+    testImplementation 'junit:junit:4.13'
+    testImplementation 'org.mockito:mockito-core:3.5.13'
```

Comment:
@rafalgawlik wygląda na to, że chodzi o ten update :man_facepalming: 
zostawię `testImplementation 'org.mockito:mockito-core:1.+'`